opentelemetry-semantic-conventions/src/opentelemetry/semconv/trace/__init__.py

```diff
@@ -608,7 +608,7 @@ class SpanAttributes:
     MESSAGING_KAFKA_MESSAGE_KEY = "messaging.kafka.message_key"
     """
     Message keys in Kafka are used for grouping alike messages to ensure they're processed on the same partition. They differ from `messaging.message_id` in that they're not unique. If the key is `null`, the attribute MUST NOT be set.
-    Note: If the key type is not string, it's string representation has to be supplied for the attribute. If the key has no unambiguous, canonical string form, don't include its value.
+    Note: If the key type is not string, its string representation has to be supplied for the attribute. If the key has no unambiguous, canonical string form, don't include its value.
```

Comment:
```suggestion
    Note: If the key type is not `string`, its string representation has to be supplied for the attribute. If the key has no unambiguous, canonical string form, do not include its value.
```

Just a passing by comment about some further doc wins I think we could get 🙂

Reply:
The monospace formatting on `string` makes sense.

For “don't”→“do not”, are you trying to discourage contractions in the documentation as a matter of style? They are currently rather widely used.

Reply:
Yes, I just have a personal custom to discourage contractions in documentation, but it's not a SIG requirement so it's up to you!

Reply:
OK, how would you feel about me introducing those two changes (and replacing other contractions at the same time) in a follow-up PR instead?

Reply:
Definitely okay with you leaving out the contraction change, but truthfully I wouldn't go through the effort of changing contractions everywhere because other members in the SIG might disagree with me. We can discuss at the SIG meeting next week if that's something people want, or you could raise an issue first for visibility in case and gauge interest from the community 🙂